src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -36,40 +45,104 @@ In development mode, React will rethrow errors caught within an error boundary.
-      Sentry.withScope((scope) => {
-          scope.setExtras(errorInfo);
-          const eventId = Sentry.captureException(error);
-          this.setState({eventId});
-      });
-    }
-
-    render() {
-        if (this.state.hasError) {
-            //render fallback UI
-            return (
-              <button onClick={() => Sentry.showReportDialog({ eventId: this.state.eventId })}>Report feedback</button>
-            );
-        }
-
-        //when there's not an error, render children untouched
-        return this.props.children;
-    }
+import React from 'react';
+import * as Sentry from '@sentry/react';
+
+import {Example} from '../example';
+
+function FallbackComponent() {
+  return (
+    <div>An error has occured</div>
+  )
+}
+
+class App extends React.Component {
+  render() {
+    return (
+      <Sentry.ErrorBoundary fallback={FallbackComponent} showDialog>
+        <Example />
+      </Sentry.ErrorBoundary>
+    )
+  }
 }
 
-export default ExampleBoundary
+export default App;
 ```
+
+### ErrorBoundary Options
+
+The ErrorBoundary component exposes a variety props that can be passed in for extra configuration. There are no required options, but it is highly recommended that a fallback component be set.
+
+#### `showDialog` (boolean)
+
+If a [Sentry User Feedback Widget]({%- link _documentation/enriching-error-data/user-feedback.md -%}?platform={{ include.platform }}) should be rendered when the Error Boundary catches an error.
+
+#### `dialogOptions` (Object)
+
+Options that are passed into the Sentry User Feedback Widget. See all possible customization options [here]({%- link _documentation/enriching-error-data/user-feedback.md -%}?platform={{ include.platform }}#customizing-the-widget)
+
+#### `fallback` (React.ReactNode or Function)
+
+A fallback component that gets rendered when the error boundary encounters an error. Can either provide a React Component, or a function that returns React Component as a valid fallback prop. If a function is provided, the function will be called with the error and the component stack at time of error, and a function that resets the error boundary to it's regular state.
+
+#### `onError` (Function)
+
+A function that gets called when the Error Boundary encounter an error. `onError` is useful if you want to propogate the error into a state management library like Redux, or if you want to check any side effects that could have occured due to the error.
+
+#### `onMount` (Function)
+
+A function that gets called on ErrorBoundary `componentDidMount()`
+
+#### `onUnmount` (Function)
+
+A function that gets called on ErrorBoundary `componentWillUnmount()`
+
+### ErrorBoundary Usage
+
+#### Setting a Fallback Function (Render Props)
+
+Below is an example where a fallback prop, using the [render props approach](https://reactjs.org/docs/render-props.html), is used to display a fallback UI on error, and gracefully return to a normal component state when reset.
```

Comment:
```suggestion
Below is an example where a fallback prop, using the [render props approach](https://reactjs.org/docs/render-props.html), is used to display a fallback UI on error, and gracefully return to a standard component state when reset.
```

We try to avoid the word "normal" because it's not inclusive language.

Reply:
Ok will keep that in mind 👍